sonar-plugin-api/src/main/java/org/sonar/api/test/Testable.java

```diff
@@ -19,12 +19,15 @@
  */
 package org.sonar.api.test;
 
-import org.sonar.api.component.Perspective;
-
 import java.util.List;
 import java.util.Map;
 import java.util.SortedSet;
+import org.sonar.api.component.Perspective;
 
+/**
+ * @deprecated since 5.2
+ */
+@Deprecated
 public interface Testable extends Perspective {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png)Do not forget to remove this deprecated code someday.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)